### PR TITLE
Update hu.po

### PR DESCRIPTION
**Pull request by ovari**
https://github.com/tchx84/Flatseal/issues/9

Please advise:
1. When we can delete our fork
2. When the version with hu support will be available on [Flathub](https://flathub.org/apps/details/com.github.tchx84.Flatseal)

Thank you

### CODE REVIEW

File: po/hu.po [hunk 1]
```diff
@@ -20,132 +20,133 @@ msgstr ""
...
 
 #: src/model.js:65
 msgid "Access D-Bus system bus (unrestricted)"
-msgstr ""
+msgstr "A D-Bus rendszer busz hozzáférése (korlátlan)"
 
 #: src/model.js:70
 msgid "Access D-Bus session bus (unrestricted)"
-msgstr ""
+msgstr "A D-Bus rendszer busz hozzáférése (korlátlan)"
```
Comment (tchx84) [inline po/hu.po]: Can you make it clear that this permission is for D-Bus session bus?

File: po/hu.po [hunk 2]
```diff
@@ -20,132 +20,133 @@ msgstr ""
...
 
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:29
 msgid "Martin Abente Lahaye"
-msgstr ""
+msgstr "Lahaye Martin Abente"
 
 #: src/aboutDialog.ui:11
 msgid "Copyright © 2020 - Martín Abente Lahaye"
-msgstr ""
+msgstr "Szerzői jog © 2020 - Lahaye Martín Abente"
```
Comment (tchx84) [inline po/hu.po]: You can leave this one out.

File: po/hu.po [hunk 3]
```diff
@@ -20,132 +20,133 @@ msgstr ""
...
 "Flatseal is a graphical utility to review and modify basic permissions from "
 "your Flatpak applications."
-msgstr ""
+msgstr "A Flatseal egy grafikus segédprogram, amely áttekintheti és "
+"módosíthatja a Flatpak-alkalmazások alapvető engedélyeit."
 
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:29
 msgid "Martin Abente Lahaye"
-msgstr ""
+msgstr "Lahaye Martin Abente"
```
Comment (tchx84) [inline po/hu.po]: You can leave this one out.

File: po/hu.po [hunk 4]
```diff
@@ -20,132 +20,133 @@ msgstr ""
...
 #: data/com.github.tchx84.Flatseal.desktop.in:9
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:8
 msgid "A permissions manager for Flatpak"
-msgstr ""
+msgstr "A Flatpak engedélykezelője"
 
 #: data/com.github.tchx84.Flatseal.desktop.in:10
 msgid "com.github.tchx84.Flatseal"
-msgstr ""
+msgstr "com.github.tchx84.Flatseal"
```
Comment (tchx84) [inline po/hu.po]: You can this one out.

**Review by tchx84 — CHANGES_REQUESTED** [review]
Just a few nitpicks, and one comment regarding session bus.

File: po/hu.po (hunk 1 above)
Comment (ovari) [reply po/hu.po]: Done

File: po/hu.po (hunk 3 above)
Comment (ovari) [reply po/hu.po]: Done

File: po/hu.po (hunk 2 above)
Comment (ovari) [reply po/hu.po]: Done

File: po/hu.po (hunk 4 above)
Comment (ovari) [reply po/hu.po]: Done

**Comment by ovari** [comment]
Feel free to change any of our suggestions.

File: po/hu.po [hunk 5]
```diff
@@ -20,12 +20,12 @@ msgstr ""
 #: data/com.github.tchx84.Flatseal.desktop.in:3
 #: data/com.github.tchx84.Flatseal.appdata.xml.in:3 src/aboutDialog.ui:14
 msgid "Flatseal"
-msgstr ""
+msgstr "Flatseal"
```
Comment (tchx84) [inline po/hu.po]: Ops, this one should be left out as well.

**Review by tchx84 — CHANGES_REQUESTED** [review]
Almost ready, just one last nitpick.

File: po/hu.po (hunk 5 above)
Comment (ovari) [reply po/hu.po]: Done. If there are any more, you are more than welcome to change them. It is your program after all.

File: po/hu.po [hunk 6]
```diff
@@ -47,105 +48,105 @@ msgstr ""
...
 
 #: src/model.js:70
 msgid "Access D-Bus session bus (unrestricted)"
-msgstr ""
+msgstr "A D-Bus munkamenet busz hozzáférése (korlátlan)"
 
 #: src/model.js:75
 msgid "Access Secure Shell agent"
-msgstr ""
+msgstr "A biztonsági héj ügynök hozzáférése"
```
Comment (tchx84) [inline po/hu.po]: Should use "Secure Shell" here.

**Review by tchx84 — CHANGES_REQUESTED** [review]
Noticed one more thing while testing (see comment above), plus needs a rebase.

File: po/hu.po (hunk 6 above)
Comment (ovari) [reply po/hu.po]: Done. If there is anything else like this, you are welcome to change. Don't understand why there is so much back and forth.

Don't know what and how to do a rebase. Can you please do that?

Thank you

**Comment by ovari** [comment]
@tchx84 can we delete our fork? Thank you

**Comment by tchx84** [comment]
If you could do the rebase it would be great, but if you don't have any time left, feel free to do what you consider best. Thanks.

**Comment by ovari** [comment]
Can you please *rebase*? Not sure how to rebase using the web interface. Or do we need to make this file in the new version? Thank you

**Comment by tchx84** [comment]
Yes, It would probably be easier to just start from a new file and move your translations to it. You can create a new PR if that makes it easier for you, as well.

**Comment by ovari** [comment]
Done. Can you please accept https://github.com/tchx84/Flatseal/pull/23 ?

**Pull request closed without merging**